lib/crypto/src/elliptic-curve/projective.rs

```diff
@@ -0,0 +1,326 @@
+//! Projective curve points.
+use core::{
```

Comment:
Make sense to `projective` and `afine` into `point` module. Wdt?

Reply:
I think `arithmetic` module also can be put there

Reply:
Makes sense 👍